submissions/cumiOyemike/ADS/index.html

```diff
@@ -7,7 +7,11 @@
     <title>Document</title>
```

Comment:
Also, this file should be inside the javascript folder or HTML folder

Reply:
I don't understand ma. These are "cumi oyemike's" files.